### DISCUSSION TITLE
problem with Xiaomi LLKZMK11LM

### DISCUSSION THREAD

**Discussion by vane-666** [Koenkk/zigbee2mqtt#19900] [Devices]
Good afternoon. There are six Xiaomi LLKZMK11LM relays. They control the lighting; switches and lamps are connected to them.
Some problems are observed; at random moments the relays may turn the lighting on or off themselves. At the same time, many events are also registered - Power outage count changes the state.
Tell me what could be the matter? It seems that there is some kind of problem in the firmware of the relay or in the relays themselves, as far as I understand Power outage count is when the power to the relay is lost, but the electricity does not turn off. Perhaps the relays react somehow strangely to the voltage in the network.
![Снимок экрана 2023-11-28 093458](https://github.com/Koenkk/zigbee2mqtt/assets/68059472/acaf9e79-78c0-41e2-a884-9b1716f99e36)

**Comment by ReX1983** [comment]
I have the same issue. Did you sort it out?

Mine keeps restarting every hour or so, claiming there is a power outage, but it is not the case.

**Reply by vane-666** [reply]
I reinstalled zigbee2mqtt and reconfigured everything. Got better

**Reply by ReX1983** [reply]
> I reinstalled zigbee2mqtt and reconfigured everything. Got better

Thanks. So, it doesn't reboot randomly anymore?

**Reply by vane-666** [reply]
occasionally reboots

**Reply by vaidasL** [reply]
been struggling with the same issue for awhile (3 devices ATM). What finally worked is making sure the relays are not connected directly to the coordinator.

coordinator: CC2531
firmware: 20190619

hope that helps others as well.

**Reply by ReX1983** [reply]
> been struggling with the same issue for awhile (3 devices ATM). What finally worked is making sure the relays are not connected directly to the coordinator.
> 
> coordinator: CC2531 firmware: 20190619
> 
> hope that helps others as well.

Interesting.

In my case it reboots if it is used as repeater for Xiaomi end devices, like the temperature sensor.

Indeed I have noted that the Xiaomi temperature sensor drops off from the network is used with other repeaters, so it seems that Xiaomi hacked a solution, by forcing its own repeaters to reboot in order to keep connections to the problematic end devices.

**Reply by makskhats** [reply]
Were you able to solve the problem? I have over 20 relays and absolutely all of them behave in a similar way. I also observe periodic changes in the "outage count" and, if the external button is connected in the on position, this is accompanied by a short-term switching on of the light.
I see that you partially solved the problem by reconfiguring the z2m. But I observe this phenomenon even when they are working in autonomous mode.
I also see an interesting assumption about a failure due to operation in repeater mode, but as I understand it, it cannot be turned off.
But, as far as I remember, this problem existed even when the device was in autonomous mode, not connected to any hub.

**Reply by ReX1983** [reply]
> Were you able to solve the problem? I have over 20 relays and absolutely all of them behave in a similar way. I also observe periodic changes in the "outage count" and, if the external button is connected in the on position, this is accompanied by a short-term switching on of the light. I see that you partially solved the problem by reconfiguring the z2m. But I observe this phenomenon even when they are working in autonomous mode. I also see an interesting assumption about a failure due to operation in repeater mode, but as I understand it, it cannot be turned off. But, as far as I remember, this problem existed even when the device was in autonomous mode, not connected to any hub.

I think it is programmed to reboots if used as repeater. You can only avoid it by not pairing any end device to it and actively avoiding that they connect to it. I have not investigated further as it seems an intended (although annoying) behaviour.

**Reply by vaidasL** [reply]
An update after 2 months: my approach described above proved to be incomplete - avoiding direct connections to coordinator looked promising at first (and seemed to be working!), but after a week or so one of the relays went back to the old behavior. Not that bad as before, but still annoying, especially when it controls lights.
So I gave up: now those relays are living their happy life in a separate zigbee network and finally with 0 issues. 

![IMG_5391](https://github.com/user-attachments/assets/91d760f2-b3fa-4448-9824-b8f6bbb88347)

I was lucky to have big enough network so that removal of those relays didn’t affect the coverage and stability of original mesh. 

Btw, I could not get two zigbee usb sticks to work reliably in a single machine, ended up with a separate rpi device and remote adapter setup

**Reply by makskhats** [reply]
This is the most adequate version I found. Of course, it is terrible that Aqara made such a blunder. It is unbearable and judging by everything I will be forced to get rid of either all Aqara relays or all Aqara temperature sensors (I hope that the problem is only with temperature sensors). A very unpleasant situation, since there are a lot of these relays and these sensors in the house. Splitting into two networks would be a good solution, but splitting into two RPIs is too much. Thank you very much for these comments.
